src/integration.koa.test.ts

```diff
@@ -151,6 +151,9 @@ test('GET method', async () => {
     {
       implementation: {
         'GET /posts': (ctx) => {
+          // TypeScript isn't smart enough to determine the correct ctx.request
+          // property here.
+          // @ts-ignore
```

Comment:
wanna put `@ts-expect-error` just in case this is figured out in the future? then TS will tell us that this comment isn't necessary anymore

Reply:
Good idea.